spec/factories/articles.rb

```diff
@@ -3,5 +3,6 @@
     title { "MyString" }
     body { "MyText" }
     author { "MyString" }
+    association :user
```

Comment:
Again, why is `author` a string? Or rather, do we need that at all, or should the association with the user be modified. I know that Oliver already commented on this, but to be clear, this is a huge blocker.